third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -433,35 +433,71 @@ static Value convertBf16ToFp32(Location loc,
   return bitcast(shifted, f32_ty);
 }
 
+static Value buildGCNInstruction(Location loc,
+                                 ConversionPatternRewriter &rewritter,
+                                 const std::string &instr_name,
```

Comment:
Nit: This is not following naming convention; should be `instrName`. Also we can use `StringRef` as the type here.